packages/web/src/shared/components/ui/link/index.tsx

```diff
@@ -1,15 +1,26 @@
+import clsx from 'clsx'
 import ReactLink from 'next/link'
 
+import styles from '@/styles/classes'
+
 interface Props {
   text: string
-  link: string
+  href: string
+  buttonStyle?: 'primary' | 'secondary'
   className?: string
 }
 
-export function Link({ text, link, className }: Props) {
+export function Link({ text, href, buttonStyle, className }: Props) {
   return (
-    <ReactLink href={link}>
-      <span className={`text-yellow font-normal hover:underline ${className}`}>
+    <ReactLink href={href}>
+      <span
+        className={clsx(className, {
+          'text-yellow hover:underline': !buttonStyle,
+          [styles.button]: !!buttonStyle,
```

Comment:
sugestão: se for pra jogar os estilos num variável, faça isso com todos os casos, senão fica confuso de ler. Eu também evitaria ao máximo esse `!!` em favor do `Boolean()` que é um pouco mais semântico, mas é bastante preferência isso. Fiz esse comentário pq quase abri outro perguntando se era intencional o `!!`, mas daí eu vi o caso de cima e entendi o que tava acontecendo.